Makefile

```diff
@@ -1,14 +1,18 @@
+include ./envs/versions.env
+export
+export GH_TOKEN=
```

Comment:
Just note (to myself mostly):
Line 2 is necessary I guess to access these env variables in the docker make commands.
Line 3 is just workaround I guess for the gh login command, and it is not possible to get rid of it.